app.js

```diff
@@ -10,10 +10,94 @@ app.locals.title = 'Test Express';
 app.use(cors());
 app.use(express.json());
 
-// All endpoints live here
+app.get('/api/v1/movies', async (req, res) => {
+  try {
+    const movies = await database('movies').select();
+    res.status(200).json(movies);
+  } catch(e) {
+    res.status(500).json({e})
+  }
+})
 
-app.get('/', (request, response) => {
-  response.send('We\'re going to test all the routes!');
-});
```

Comment:
We determined that this test would not pass unless we added a dummy endpoint at `('/')` which we ultimately determined was unnecessary.

Reply:
Thanks for adding this explanation ^